package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "slice",
   "displayName": "Slice",
-  "description": "Provides support for Slice syntax highlighting.",
+  "description": "Provides a Slice Language Server and support for Slice syntax highlighting.",
```

Comment:
I'd make it simpler. If a user wants to know the list of features, they can read the full text.
If it's too long, it just cuts off with `...`s.

For example, here's some other extension's descriptions:

- Rust language support Visual Studio Code
- All-in-One PHP support
- Ruby language support and debugging for Visual Studio Code